codegen-server/python/src/main/kotlin/software/amazon/smithy/rust/codegen/server/python/smithy/generators/PythonServerAttributeUtils.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.rust.codegen.server.python.smithy.generators
+
+import software.amazon.smithy.rust.codegen.rustlang.RustWriter
+import software.amazon.smithy.rust.codegen.rustlang.asType
+import software.amazon.smithy.rust.codegen.rustlang.rustTemplate
+import software.amazon.smithy.rust.codegen.server.python.smithy.PythonServerCargoDependency
+
+/**
+ * This module contains utilities to render PyO3 attributes.
```

Comment:
Are these extension methods really necessary? It think you get the same syntax sugar by using the `Attribute` class from `RustTypes` and calling `render`.

Reply:
You are probably right, but I think having these methods in their own separate file is actually nice and I think this file will grow over time with new helpers.. If don't have strong opinions on this, if you feel we should get rid of them I can try to just use the `Attribute` class.

Reply:
It looks like the issue comes from https://github.com/awslabs/smithy-rs/blob/15e1b3425f56ff2606121e99dff11de38424c744/codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/rustlang/RustTypes.kt#L377

This line uses `writer.raw()`. If I change the line from `writer.raw("#$bang[$annotation]")` to `writer.rust("##$bang[$annotation]"` it start rendering as I would expect.

I am not sure of the implications of making this change.. @rcoh WDYT?

Reply:
https://github.com/awslabs/smithy-rs/issues/1465.